.github/workflows/main.yml

```diff
@@ -7,11 +7,11 @@ env:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        rust:
-          - stable
+        os: [ubuntu-latest, macos-latest]
```

Comment:
I think we should hardcode macos version, because macOS 14 to 15 update will definitely break our CI.

Reply:
Indeed.